examples/hybrid/staggered_nonhydrostatic_model.jl

```diff
@@ -10,7 +10,7 @@ const TD = Thermodynamics
 include("schur_complement_W.jl")
 include("hyperdiffusion.jl")
 
-f_plane_coriolis_frequency(::AbstractEarthParameterSet) = 0
+f_plane_coriolis_frequency(::AbstractEarthParameterSet) = FT(0)
```

Comment:
Let's leave this as it was, we need to wrap calls to parameter set functions with type casting anyway.

Reply:
Ok, I moved the `FT()` call from inside this function to where this function is called.